roles/st2chatops/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+# st2chatops version to install. Use `latest` to get automatic updates or pin it to numeric version like `2.2.0`
+st2chatops_version: latest
+
+# hash to configure any value in st2chatops.env
+# https://github.com/StackStorm/st2chatops/blob/master/st2chatops.env
+st2chatops_config:
+  ST2_API_KEY: CHANGE-ME-PLEASE
+  HUBOT_ADAPTER: slack
```

Comment:
I think we shouldn't set `HUBOT_ADAPTER: slack` by default if user didn't provide any custom `key: value` for that.
WDYT?